lang/cs/Org.Apache.REEF.Evaluator.Tests/TaskRuntimeTests.cs

```diff
@@ -214,6 +215,92 @@ public void TestTaskEventsOnFailure()
             Assert.Equal(testTaskEventStopHandler.StopInvoked.Value, taskId);
         }
 
+        /// <summary>
+        /// Tests that suspend ends the task and invokes the right handler.
+        /// </summary>
+        [Fact]
+        public void TestSuspendTask()
+        {
+            var contextId = Guid.NewGuid().ToString();
+            var taskId = Guid.NewGuid().ToString();
+
+            var injector = GetInjector(typeof(CountDownAction), contextId, taskId);
+            var taskRuntime = injector.GetInstance<TaskRuntime>();
+            taskRuntime.RunTask();
+
+            var task = injector.GetInstance<TestTask>();
```

Comment:
You might want to bind TestTask to ITask and bind TestTask to TaskConfigurationOptions.SuspendHandler, then inject ITask to make the test case clear for what you are using for SuspendHandler explicitly.